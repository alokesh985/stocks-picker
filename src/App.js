// Components
import StockPicker from "./components/stockPicker";

function App() {
  return <StockPicker />;
}

export default App;
