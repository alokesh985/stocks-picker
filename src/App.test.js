import renderer from "react-test-renderer";
import App from './App';

test("Should match snapshot", () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
