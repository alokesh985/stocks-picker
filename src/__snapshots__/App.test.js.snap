// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<div>
  <div
    className="appContainer"
  >
    <h1
      className="header"
    >
      Stock Picker
    </h1>
    <div
      className="autoCompleteContainer"
    >
      <div
        className="searchContainer"
      >
        <input
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          placeholder="Enter stock name here"
          type="text"
          value=""
        />
        <button
          className="searchButton"
          onClick={[Function]}
        >
          <img
            alt="search-icon"
            className="searchIcon"
            src="search-icon.svg"
          />
        </button>
      </div>
    </div>
    <div
      className="container"
    >
      <h2>
        No Stocks Selected!
      </h2>
      <h2>
        Please search for a stock on the search bar above
      </h2>
    </div>
    <div
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      style={
        Object {
          "bottom": 16,
          "left": 16,
          "pointerEvents": "none",
          "position": "fixed",
          "right": 16,
          "top": 16,
          "zIndex": 9999,
        }
      }
    />
  </div>
</div>
`;
